test/jdk/sun/security/tools/keytool/fakegen/java.base/sun/security/rsa/RSAKeyPairGenerator.java

```diff
@@ -57,7 +57,7 @@ public void initialize(int keySize, SecureRandom random) {
             initialize(new RSAKeyGenParameterSpec(keySize,
                     RSAKeyGenParameterSpec.F4), random);
         } catch (InvalidAlgorithmParameterException iape) {
-            throw new InvalidParameterException(iape.getMessage());
+            throw new InvalidParameterException(iape.getMessage(), iape);
```

Comment:
Use the one-arg ctor to make this consistent with the code above: [src/java.base/share/classes/sun/security/rsa/RSAKeyPairGenerator.java](https://github.com/openjdk/jdk/pull/10966/files#diff-4eaff4fafb2999bb74081347365803821458dc8ea618808474a948a4dc7d9b75)